apps/web/components/dashboard/preview/BookmarkHtmlHighlighter.tsx

```diff
@@ -1,101 +1,25 @@
-import React, { useEffect, useRef, useState } from "react";
-import { ActionButton } from "@/components/ui/action-button";
-import { Button } from "@/components/ui/button";
-import { Popover, PopoverContent } from "@/components/ui/popover";
+import React, { useCallback, useEffect, useRef, useState } from "react";
+import { ColorPickerMenu } from "@/components/dashboard/preview/ColorPickerMenu";
+import { toast } from "@/components/ui/use-toast";
```

Comment:
this is unused. there are some other lint/type errors in the branch, can you take a look and address them?